.travis.yml

```diff
@@ -24,6 +22,11 @@ install:
 
 script:
   - tox
-  - if [[ $COVERAGE_COVERAGE == 'yes' ]]; then python igor.py combine_html; fi
-  - if [[ $COVERAGE_COVERAGE == 'yes' ]]; then pip install codecov; fi
-  - if [[ $COVERAGE_COVERAGE == 'yes' ]]; then codecov -X gcov --file coverage.xml; fi
+
+after_script:
+  - |
+    if [[ $COVERAGE_COVERAGE == 'yes' ]]; then
+      python igor.py combine_html
+      pip install codecov
+      codecov -X gcov coveragepy search xcode fix -f coverage.xml -n $TOXENV
```

Comment:
What do these arguments do?

Reply:
The disable more checks, e.g. "coveragepy" is not necessary, since there is a coverage.xml already, and the others are mainly for performance.
But I suspect that "fix" might have an influence on "def"s being covered or not.